### PR TITLE
Don't use raw identifiers unless necessary

### PR DESCRIPTION
**Pull request by rylev**
This makes looking at docs much easier since there's not a bunch of `r#` everywhere. I also cleaned up checking `this` pointers for null.

### CODE REVIEW

File: crates/gen/src/types/method.rs [hunk 1]
```diff
@@ -230,29 +230,21 @@ impl Method {
...
-                    match self.ptr.get_abi() {
-                        None => panic!("The `this` pointer was null when calling method"),
-                        Some(this) => {
-                            unsafe {
-                                let mut __ok: #return_type = ::std::mem::zeroed();
-                                (this.vtable().#method_name)(this, #args #return_arg)
-                                    .and_then(|| __ok )
-                            }
-                        }
+                    let this = self.ptr.get_abi().expect("The `this` pointer was null when calling method");
```
Comment (Josh015) [inline crates/gen/src/types/method.rs:233]: Can't this be condensed down to "self.get_abi()" now?

File: crates/gen/src/types/method.rs [hunk 2]
```diff
@@ -230,29 +230,21 @@ impl Method {
...
                 pub fn #method_name<#constraints>(&self, #params) -> ::winrt::Result<()> {
                     use ::winrt::AbiTransferable;
-                    match self.ptr.get_abi() {
-                        None => panic!("The `this` pointer was null when calling method"),
-                        Some(this) => {
-                            unsafe {
-                                (this.vtable().#method_name)(this, #args).ok()
-                            }
-                        }
+                    let this = self.ptr.get_abi().expect("The `this` pointer was null when calling method");
```
Comment (Josh015) [inline crates/gen/src/types/method.rs:245]: Can't the "self.ptr.get_abi()" be condensed down to "self.get_abi()" now?

File: crates/gen/src/types/method.rs (hunk 1 above)
Comment (rylev) [reply crates/gen/src/types/method.rs:233]: Yes it can. I can push that real quick.

**Comment by Josh015** [comment]
@rylev Don't forget to add ".expect()" to "activation_factory.rs".

**Pull request merged**